tests/Microsoft.Identity.Test.Unit/Helpers/ParallelRequestMockHanler.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.Http;
+using Microsoft.Identity.Client.OAuth2;
+using Microsoft.Identity.Test.Common.Core.Mocks;
+using Microsoft.Identity.Test.Unit.RequestsTests;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Identity.Test.Unit.Helpers
+{
+    /// <summary>
+    /// This custom HttpManager does the following: 
+    /// - provides a standard response for discovery calls
+    /// - responds with valid tokens based on a naming convention (uid = "uid" + rtSecret, upn = "user_" + rtSecret)
+    /// </summary>
+    internal class ParallelRequestMockHanler : IHttpManager
```

Comment:
Seems like you have 2 files that do very similar things.
`ParallelRequestMockHandler` and `ParallelRequestMockHanler` 
Was one of these supposed to replace the other?

Reply:
Yes, thx. Merge error. Fixed now.